### PR TITLE
Create api demo for rapid api #5

### PR DESCRIPTION
**Pull request by Tojonirina4**
I created 3 examples
1. Simple get
2. get with required params
3. get with required params and optional query

### CODE REVIEW

**Review by gbudjeakp — APPROVED** [review]

File: server/routes/flight.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+const express = require("express");
+const router = express.Router();
+const unirest = require("unirest");
+const rapidApiKey = "b45bedc57emsh23d534a09704e52p1ab1a8jsn764974156db7";
+const rapidApiHost = "skyscanner-skyscanner-flight-search-v1.p.rapidapi.com";
```
Comment (johnsciutto) [inline server/routes/flight.js]: I suggest putting this information in the `.env` file.

**Review by johnsciutto — CHANGES_REQUESTED** [review]

File: server/app.js [hunk 2]
```diff
@@ -6,8 +6,10 @@ const logger = require("morgan");
 
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
+const flightRouter = require("./routes/flight");
 
 const { json, urlencoded } = express;
+const port = process.env.port || 3001;
```
Comment (sina-jamshidi) [inline server/app.js]: shouldn't be necessary

File: server/app.js [hunk 3]
```diff
@@ -36,4 +39,8 @@ app.use(function(err, req, res, next) {
   res.json({ error: err });
 });
 
+app.listen(port, () => {
+  console.log(`Server is listening at http://localhost:${port}`);
+})
+
```
Comment (sina-jamshidi) [inline server/app.js]: not necessary

**Review by sina-jamshidi — APPROVED** [review]
This should be fine once you remove the port and figure out the `npm run dev`

**Review by johnsciutto — APPROVED** [review]

**Pull request merged**